demos/asp-net-core/ITOps.ViewModelComposition/DynamicViewModel.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Engine
```

Comment:
As per https://github.com/Particular/Workshop.Microservices/pull/59#issuecomment-318675121, should this be moved to `ITOps.ViewModelComposition`?

Reply:
done